metricflow-semantics/metricflow_semantics/model/linkable_element_property.py

```diff
@@ -27,6 +27,8 @@ class LinkableElementProperty(Enum):
     METRIC_TIME = "metric_time"
     # Refers to a metric, not a dimension.
     METRIC = "metric"
+    # An time dimension with a DatePart.
```

Comment:
nit: I think you meant "A" not "An" here, right?  (Or is this a convention I'm not aware of?)